### PR TITLE
Adding airgap install instructions for existing clusters

### PR DESCRIPTION
**Pull request by divolgin**

### CODE REVIEW

File: content/kotsadm/installing/airgap-packages.md [hunk 1]
```diff
@@ -8,6 +8,34 @@ draft: false
 
 When installing an application from an airgap package, the container images and application manifests are provided by the application vendor in an archive that can be used to deliver the artifacts into the cluster. This feature is only available for licenses that have the airgap feature enabled.
 
+## Kots install
+Begin by [installing the Kots CLI kubectl plugin](/kots-cli/getting-started/).  The Admin Console can be installed using the CLI and the airgap package included in the [release assets](https://github.com/replicatedhq/kots/releases) named `kotsadm.tar.gz`.   The asset version has to match the CLI version, which can be determined by running:
```
Comment (marccampbell) [inline content/kotsadm/installing/airgap-packages.md:12]: Begin by [installing the KOTS kubectl plugin](/kots-cli/getting-started/). 
The Admin Console can be installed using the KOTS plugin and the airgap package that can be downloaded from the [release assets](https://github.com/replicatedhq/kots/releases/latest) named `kotsadm.tar.gz`. 
The asset version must match the KOTS CLI version, which can be determined by running:

File: content/kotsadm/installing/airgap-packages.md [hunk 2]
```diff
@@ -8,6 +8,34 @@ draft: false
...
+
+```shell
+kubectl kots version
+```
+
+First step is to extract kotsadm images and push them into a private registry.  Registry credentials provided in this step need to have push access.  These credentials will not be stored anywhere.
```
Comment (marccampbell) [inline content/kotsadm/installing/airgap-packages.md:18]: The first step is to extract KOTS Admin Console container images and push them into a private registry.
Registry credentials provided in this step must have push access.
These credentials will not be stored anywhere or reused later.

File: content/kotsadm/installing/airgap-packages.md [hunk 3]
```diff
@@ -8,6 +8,34 @@ draft: false
...
+
+```shell
+kubectl kots admin-console push-images ./kotsadm.tar.gz private.registry.host/kotsadm --registry-username rw-user --registry-password rw-password
```
Comment (marccampbell) [inline content/kotsadm/installing/airgap-packages.md:21]: kubectl kots admin-console push-images ./kotsadm.tar.gz private.registry.host/application-name \
    --registry-username rw-username \
    --registry-password rw-password

File: content/kotsadm/installing/airgap-packages.md [hunk 4]
```diff
@@ -8,6 +8,34 @@ draft: false
...
+```
+
+The next step is to install the Admin Console using images pushed in the previous step.  Registry credentials provided in this step only need to have read access, and they will be stored in a kubernetes secret in the same namespace where Admin Console will be installed.
```
Comment (marccampbell) [inline content/kotsadm/installing/airgap-packages.md:24]: stored in a Kubernetes secret in the same namespace where Admin Console will be installed.
These credentials will be used to pull the images, and will be automatically created as an imagePullSecret on all of the Admin Console pods.

File: content/kotsadm/installing/airgap-packages.md [hunk 5]
```diff
@@ -8,6 +8,34 @@ draft: false
...
+
+```shell
+kubectl kots install --kotsadm-registry private.registry.host/kotsadm --registry-username ro-user --registry-password ro-password application-name
```
Comment (marccampbell) [inline content/kotsadm/installing/airgap-packages.md:27]: kubectl kots install \
    --kotsadm-registry private.registry.host/application-name \
    --registry-username ro-username \
    --registry-password ro-password \
    application-name

File: content/kotsadm/installing/airgap-packages.md [hunk 6]
```diff
@@ -8,6 +8,34 @@ draft: false
...
+```
+
+Once this has completed, the kots plugin will create a port-forward to the Admin Console interface. The Admin Console API and Web server are exposed over a ClusterIP service in the namespace provided. The port-forward will be active as long as the CLI is running. Pressing Ctrl+C will end the port forward.
```
Comment (marccampbell) [inline content/kotsadm/installing/airgap-packages.md:30]: Once this has completed, the KOTS will create a port-forward to the Admin Console on port 8800. 
The Admin Console is exposed internally in the cluster, and can only be accessed using a port forward.
The port-forward will be active as long as the CLI is running. 
Pressing Ctrl+C will end the port forward.

File: content/kotsadm/installing/airgap-packages.md [hunk 7]
```diff
@@ -8,6 +8,34 @@ draft: false
...
+
+```shell
+  • Press Ctrl+C to exit
+  • Go to http://localhost:8800 to access the Admin Console
+```
+
+Once this message is printed in the terminal, click the link, or visit `http://localhost:8800` to complete the setup using the Admin Console web-based UI.
```
Comment (marccampbell) [inline content/kotsadm/installing/airgap-packages.md:37]: Once this message is displayed visit `http://localhost:8800` to complete the application setup using the Admin Console.

File: content/kotsadm/installing/airgap-packages.md [hunk 8]
```diff
@@ -8,6 +8,49 @@ draft: false
 
 When installing an application from an airgap package, the container images and application manifests are provided by the application vendor in an archive that can be used to deliver the artifacts into the cluster. This feature is only available for licenses that have the airgap feature enabled.
```
Comment (austinchambers) [inline content/kotsadm/installing/airgap-packages.md:9]: Unrelated to this PR, but we could remove that second line. We're talking from the perspective of the user already having the airgap bundle, so I think we can omit this line. 

```suggestion
When installing an application from an airgap package, the container images and application manifests are provided by the application vendor in an archive that can be used to deliver the artifacts into the cluster. 
```

**Comment by divolgin** [comment]
Added a note about embedded clusters.

**Pull request merged**